tests/python_test_runner.py

```
@@ -0,0 +1,232 @@
+                        it_pos += 1
+                    
+                    test_body = describe_content[it_start:it_pos-1]
+                    
+                    tests.append({
+                        'name': full_test_name,
+                        'describe': describe_name,
+                        'it': test_name,
+                        'body': test_body.strip()
+                    })
+        
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)